src/model/OrganizationInvitation.ts

```diff
@@ -42,16 +42,39 @@ export default class OrganizationInvitation extends Ressource {
   static get(organizationId: string, id: string) {
     const { api_url } = getConfig();
 
-    return super._get<OrganizationInvitation>(`${api_url}${_url}`, { id, organizationId });
+    return super._get<OrganizationInvitation>(`${api_url}${_url}`, {
+      id,
+      organizationId,
+    });
   }
 
   static query(organizationId: string) {
     const { api_url } = getConfig();
 
-    return super._query<OrganizationInvitation>(`${api_url}${_queryUrl}`, { organizationId }, {}, {});
+    return super._query<OrganizationInvitation>(
+      `${api_url}${_queryUrl}`,
+      { organizationId },
+      {},
+      {}
+    );
+  }
+
+  static getList(organizationId: string, queryParams?: string) {
+    const { api_url } = getConfig();
+
+    return super._query<OrganizationInvitation>(
+      `${api_url}${_queryUrl}${queryParams}`,
```

Comment:
I think it could be  cleaner to remove `?` from the `queryParams` and put it directly to the interpolation. Just because that doesn't make a lot of sense to pass `?` as a parameter.